### DISCUSSION TITLE
A confusion about a mkv video which missing bit_rate information

### DISCUSSION THREAD

**Discussion by pixeldin** [mifi/lossless-cut#1582] [General]
Hello community, I try to cut a certain segment of an `input.mkv` video and this software works perfectly. 
And now I had some extra cut needed so I check some command from developer mode for reference. In a substep of cutting like this:
```
ffmpeg -hide_banner -ss '1800.00000' -i 'input.mkv' -ss 0 -t '0.79863' -map '0:0' '-c:0' h264 '-b:0' 1503233 -map '0:1' '-c:1' copy -ignore_unknown -video_track_timescale 1000 -f matroska -y 'input-smartcut-segment-encode-0.mkv' 
```
In this step, the parameter '-b:0' used as `1503233`, can someone please tell me how this value is obtained? I know that `-b:0` is an option in the FFmpeg command line parameters to set the bitrate of the video.
What I try to do is use `ffprobe` obtain the bitrate of the original video by
```
ffprobe -v error -select_streams v:0 -show_entries stream=bit_rate 
-of default=noprint_wrappers=1:nokey=1 input.mkv
```
and the output info is:
`N/A`

It seems that the `bit_rate` is not stored in the original mkv file information. If the `bit_rate` value can be calculated by other metric shown in `ffprobe`, please let me know :)

**Comment by pixeldin** [comment]
It looks fine so far, I just found the answer in the source code.
https://github.com/mifi/lossless-cut/blob/9130924e14eb66ae5e87c7b4a0a86624a52d9fd0/src/smartcut.js#L46
Thank you community, leave it here to those in need :)

**Reply by mifi** [reply]
Hi! this is because you're using "smart cut" so it will try to match the bitrate. if you disable smart cut there will be no encoding/bitrate calculation
